ansible/roles/test/vars/testcases.yml

```diff
@@ -127,6 +127,10 @@ testcases:
       filename: link_flap.yml
       topologies: [t0, t0-16, t0-64, t0-64-32, t0-116, t1, t1-lag, t1-64-lag, ptf32, ptf64]
 
+    continuous_link_flap:
+      filename: continuous_link_flap.yml
+      topologies: [t0, t0-16, t0-64, t0-64-32, t0-116, t1, t1-lag, t1-64-lag, ptf32, ptf64]
+
     mem_check:
       filename: mem_check.yml
       topologies: [t0, t0-16, t0-64, t0-64-32, t0-116, t1, t1-lag, t1-64-lag, ptf32, ptf64]
```

Comment:
>t0, t0-16, t0-64, t0-64-32, t0-116 [](start = 19, length = 34)

Why remove all the t0 topo?

Reply:
I wanted to bring only L3 interfaces down to cause BGP churn and not L2 interfaces, So made applicable only to T1. 
If we think, there is not harm to bring L2 interfaces down too, then we are good, I can add t0 topo too. Thx.

Reply:
I wanted to bring only L3 interfaces down to cause BGP churn and not L2 interfaces, So made applicable only to T1. 
If we think, there is not harm to bring L2 interfaces down too, then we are good, I can add t0 topo too. Thx.

Reply:
you can 'bring only L3 interfaces down to cause BGP churn' on T0, right?

---
In reply to: [278345351](https://github.com/Azure/sonic-mgmt/pull/810#discussion_r278345351) [](ancestors = 278345351)

Reply:
Yeah, can be done. I need to parse minigraph-facts to find only L3 interfaces and bring only those down. I wanted to do this extra exercise only, when there is some harm in bringing L2 interfaces down, else we can just use same test for T0 topo. Thx.

Reply:
Resolving, since I do not think, "sudo config interface <interface> startup" differentiate between L2 and L3 interfaces. So same test case can be used on T0. I will update testcase.yml file. Thanks.